src/udSDK/udSDK.py

```diff
@@ -1380,6 +1489,11 @@ def pointcloud(self, pointcloud: udPointCloud):
     _HandleReturnValue(self.udQueryContext_ChangePointCloud(self.pQueryContext, pointcloud.pPointCloud))
 
   def execute(self, points: udPointBufferF64):
+    """
+    Fill the point buffer with points matching the query from the pointcloud. Call repeatedly to more points matching
```

Comment:
```suggestion
    Fill the point buffer with points matching the query from the pointcloud. Call repeatedly to get more points matching
```

I'm not certain if this is the correct change, but the sentence as-is doesn't make sense.